src/components/AvatarGroup/Avatars.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { StyledAvatar } from './styled';
+import { AvatarProps, AvatarSizes } from './types';
+import { Avatar } from '..';
+
+interface Props {
+    maxAvatars?: number;
+    size?: AvatarSizes;
+    avatars?: AvatarProps[];
+}
+
+const Avatars: React.FC<Props> = (props): any => {
```

Comment:
In order to better get the benefits of typescript we should not use any type instead specify what type is returned here